core/themes/seven/css/style.css

```diff
@@ -755,16 +755,38 @@ a.button:active {
   text-decoration: none;
   color: #5a5a5a;
 }
-.node-form input#edit-submit,
-.node-form input#edit-submit-1 {
-  border: 1px solid #8eB7cd;
-  border-left-color: #8eB7cd;
-  border-right-color: #8eB7cd;
-  border-bottom-color: #7691a2;
-  background: url(../images/buttons.png) 0 -40px repeat-x;
-  color: #133B54;
+
+input.form-submit {
+  cursor: pointer;
+  padding: 10px 25px;
+  margin-bottom: 1em;
+  margin-right: 1em; /* LTR */
```

Comment:
For these margins, perhaps we could target `.form-actions input.form-submit` rather than all buttons? The assumption that all buttons need a bottom and right margin seems overly broad.